src/detect.ts

```diff
@@ -9,10 +11,22 @@ export async function detect() {
   const agent = (result ? LOCKS[path.basename(result)] : null)
 
   if (agent && !cmdExists(agent)) {
-    const url = INSTALL_PAGE[agent]
-    const link = terminalLink(url, url)
-    console.log(`Detected ${agent} but it doesn't seem to be installed.\nFollow the instructions to install ${agent}: ${link}`)
-    process.exit(1)
+    if (process.env.CI)
+      throw new Error(`Detected ${agent} but it doesn't seem to be installed`)
+
+    const link = terminalLink(agent, INSTALL_PAGE[agent])
+    console.log(`Detected ${link} but it doesn't seem to be installed.\n`)
```

Comment:
```suggestion
    console.warn(`Detected ${agent} but it doesn't seem to be installed.\n`)
    if (process.env.CI)
      process.exit(1)

    const link = terminalLink(agent, INSTALL_PAGE[agent])
```

WDYT?

Reply:
Good call! Showing the JS stack here is pretty useless